core/Rename.cpp

```diff
@@ -177,12 +176,9 @@ namespace olympia
         for(auto & i : *insts) {
             // create an index count for each instruction entered
             const auto & dests = i->getDestOpInfoList();
-            if(dests.size() > 0 && i->isStoreInst() && i->getSourceOpInfoList().size() < 2){
-                ILOG("test1");
-            }
-            if(i->isStoreInst() && i->getSourceOpInfoList().size() < 2 && dests.size() == 0){
-                // For STOREs we count for INTEGER because addresses use INTEGER PRF
-                current_counts.cumulative_reg_counts[core_types::RF_INTEGER]++;
+            if(i->isStoreInst() && dests.size() == 0){
+                // For STOREs, addresses use INTEGER PRF
```

Comment:
So do loads.  😉

Reply:
Wait, this is the dest op list.  Stores do not have dests.  This should be an assert

Reply:
Fixed in [e7fa84c](https://github.com/riscv-software-src/riscv-perf-model/pull/70/commits/e7fa84ca02af05a0422db09893bfd2815f6b2e11)